backend/src/main/twirl/MainZIOHttp4sScala3.scala.txt

```diff
@@ -21,14 +22,19 @@ object Main extends ZIOAppDefault:
     val routes = ZHttp4sServerInterpreter().from(Endpoints.all).toRoutes
     }
 
-    val port = sys.env.get("HTTP_PORT").flatMap(_.toIntOption).getOrElse(8080)
+    val port = sys.env
+      .get("HTTP_PORT")
+      .flatMap(_.toIntOption)
+      .flatMap(Port.fromInt)
+      .getOrElse(Port.fromInt(8080).get)
```

Comment:
isn't there a `port` interpolator, so we could replace this `.get` with `port"8080"`?